core/module.h

```diff
@@ -318,6 +345,13 @@ class Module {
 
   virtual CheckConstraintResult CheckModuleConstraints() const;
 
+  virtual bool is_task() { return false; };  // Whether the module has a task.
```

Comment:
Declare method as const and rename as IsTask():

`virtual bool IsTask() const { return false; };`